HW_1/task_2.py

```diff
@@ -0,0 +1,4 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+n = int(input("Введите трехзначное число: "))
+print(n // 100 + (n // 10) % 10 + n % 10)
```

Comment:
Можно еще решить через строки
```suggestion
n = input("Введите трехзначное число: ")
print(int(n[0]) + int(n[1]) + int(n[2]))
```